cmd/check_performance_counter/cmd/root.go

```diff
@@ -27,27 +31,47 @@ The defaults for this check have the --critical and --warning values set to 0, a
 retrieved is compared to be lesser than those values. Generally a counter value will be > 0, causing
 this check to emit an OK response when using these defaults.`,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmd.ParseFlags(os.Args)
+			// cmd.ParseFlags(os.Args)
```

Comment:
This supposed to be commented out?

Reply:
If we don't comment this, we got duplicates john.. attaching a screenshot for clear understanding.
![image](https://user-images.githubusercontent.com/66779959/95226575-bdb72380-081a-11eb-8d97-375427874fb2.png)